README.md

```diff
@@ -21,6 +21,16 @@ This approach supports two main use-cases:
 
 For an in depth look at the system and workflow, see the [Design](/doc/design.md#design) documentation.
 
+### Versions
```

Comment:
I don't see a link to the releases.md, we probably want that right?

Reply:
Yeah, good point, will add